### PR TITLE
Use a helper function to avoid API-breaking change

### PR DESCRIPTION
**Pull request by piotrAMD**
https://reviews.llvm.org/D72467 changes the API of ShuffleVectorInst
by replacing:

Constant *getMask()

with:

ArrayRef<int> getShuffleMask().

Temporarily use a function with an unchanged signature:

getShuffleMask(SmallVectorImpl<int> &Result)

to keep the code working before and after the LLVM upgrade.

The intention is to use getShuffleMask() eventually.

### CODE REVIEW

File: lgc/patch/llpcPatchPeepholeOpt.cpp [hunk 1]
```diff
@@ -229,8 +229,25 @@ void PatchPeepholeOpt::visitBitCast(
         insertAfter(*pBitCastRhs, *pBitCastLhs);
 
         // Create our new shuffle instruction.
+
+        // TODO: Simplify by using pShuffleVector->getShuffleMask() (no arguments) when it becomes available.
+        SmallVector<int, 8> MaskArray;
```
Comment (amdrexu) [inline lgc/patch/llpcPatchPeepholeOpt.cpp:234]: MaskArray -> maskVals

File: lgc/patch/llpcPatchPeepholeOpt.cpp [hunk 2]
```diff
@@ -229,8 +229,25 @@ void PatchPeepholeOpt::visitBitCast(
...
+        pShuffleVector->getShuffleMask(MaskArray);
+
+        auto pInt32Type = Type::getInt32Ty(pShuffleVector->getContext());
+        SmallVector<Constant*, 8> Elts;
```
Comment (amdrexu) [inline lgc/patch/llpcPatchPeepholeOpt.cpp:238]: Elts -> masks

Comment (piotrAMD) [reply lgc/patch/llpcPatchPeepholeOpt.cpp:238]: This is only a temporary change, but I will make the suggested changes anyway.

Comment (amdrexu) [reply lgc/patch/llpcPatchPeepholeOpt.cpp]: Thanks for clarification :)

**Pull request merged**